src/pages/Autolending/AutolendingWho.vue

```diff
@@ -17,30 +17,22 @@
 
```

Comment:
@eddieferrer Would appreciate if you could double check the autolending settings lightboxes to make sure I didn't mess anything up here.